assets/README.md

```diff
@@ -0,0 +1,30 @@
+Kuma Assets
+===========
+This folder contains front-end assets for developer.mozilla.org.
+
+In production, they are collected to ``/static``, processed to add
+a hash of the contents to the name, and served to users.
+In development, some may be served from this folder, to speed up
+development. See
+[Building, collecting, and serving assets](https://kuma.readthedocs.io/en/latest/assets.html)
+for the details.
+
+Here are the subfolders:
+
+* `build`: (TODO) Output of ``gulp`` and ``make compilejsi18n``, copied to
+  ``static/build``.
+* `ckeditor4`: (TODO) The source and build folders for CKEditor.
+  ``ckeditor/build`` is copied to ``static/js/libs/ckeditor/build``.
+* `dist`: (TODO) Output of the ``webpack`` process, copied to ``static/dist``.
```

Comment:
it seems a little confusing to have assets/build and assets/dist.  While we're working with two sets of build tools, maybe we could use more descriptive names like assets/gulp and assets/webpack?

(we're also going to have to figure out how to do the i18n step on the code in this directory, either with webpack or after webpack)